### DISCUSSION TITLE
How to customize error messages/alerts

### DISCUSSION THREAD

**Discussion by luigiPinnaAevo** [Budibase/budibase#8502] [Help] [answered]
Hi everyone,

I need to customize the error message when the user adds a new item to the table. 
If he violates the unique key now the system shows a typical error message like this: '_error: duplicate key value violates unique constraint "table_name"_ '. 

Is it possible to customize this error message?

Thanks

**Comment by JayP718** [comment, marked as answer]
Hi,

There is a show notification action but unfortunately there's not a way to change the default notifications.

You can read this thread here and maybe +1 https://github.com/Budibase/budibase/issues/7604 the feature to disable auto notifications. That'll at least allow you to check for the duplicates then show a custom alert.
